content/influxdb/v2.4/reference/cli/influx/config/set.md

```diff
@@ -20,15 +20,16 @@ influx config set [flags]
 `set` , `update`
 
 ## Flags
-| Flag |                  | Description                                                     | Input type | {{< cli/mapped >}}    |
-| :--- | :--------------- | :-------------------------------------------------------------- | :--------: | :-------------------- |
-| `-a` | `--active`       | Set the specified connection to active                          |            |                       |
-| `-n` | `--config-name`  | Name for the InfluxDB connection configuration to set or update |   string   |                       |
-| `-h` | `--help`         | Help for the `set` command                                      |            |                       |
-|      | `--hide-headers` | Hide table headers (default `false`)                            |            | `INFLUX_HIDE_HEADERS` |
-| `-u` | `--host-url`     | URL for InfluxDB connection configuration to set or update      |   string   |                       |
-|      | `--json`         | Output data as JSON (default `false`)                           |            | `INFLUX_OUTPUT_JSON`  |
-| `-o` | `--org`          | Organization name for the connection configuration              |   string   |                       |
+| Flag |                       | Description                                                                  | Input type | {{< cli/mapped >}}    |
+| :--- | :-------------------- | :--------------------------------------------------------------------------- | :--------: | :-------------------- |
+| `-a` | `--active`            | Set the specified connection to active                                       |            |                       |
+| `-n` | `--config-name`       | Name for the InfluxDB connection configuration to set or update              |   string   |                       |
+| `-h` | `--help`              | Help for the `set` command                                                   |            |                       |
+|      | `--hide-headers`      | Hide table headers (default `false`)                                         |            | `INFLUX_HIDE_HEADERS` |
+| `-u` | `--host-url`          | URL for InfluxDB connection configuration to set or update                   |   string   |                       |
+|      | `--json`              | Output data as JSON (default `false`)                                        |            | `INFLUX_OUTPUT_JSON`  |
+| `-o` | `--org`               | Organization name for the connection configuration                           |   string   |                       |
+| `-p` | `--username-password` | **(OSS only)** Username (and optionally password) to use for authentication. |   string   |                       |
```

Comment:
@jeffreyssmith2nd, to be sure, can you use the short flag (`-p`) to only pass the username? typically the short flags only represent one option, so wanted to verify--thanks!

Reply:
No, you can use it to pass them both (they are considered one option passed in as either `username:password` or `username`)